plugins/imagebase/plugin.js

```diff
@@ -832,6 +842,44 @@
 		this.bar.setStyle( 'width', percentage + '%' );
 	};
 
+	ProgressBar.prototype.bindEditor = function( editor ) {
+		var undoListeners = [],
+			parent = this.wrapper.getParent();
+
+		// Hide progressbar during snapshot creation so it does not trigger saving new snapshot
+		// (as content will be changed due to progressbar progress).
+		undoListeners.push(
+			editor.on( 'beforeUndoImage', function() {
+				if ( this.wrapper.getParent() ) {
```

Comment:
This function could be simplified by early return if `this.wrapper` does not have a parent.